pkg/vtpm/swtpm-vtpm/src/main.go

```diff
@@ -201,7 +207,7 @@ func handleLaunch(w http.ResponseWriter, r *http.Request) {
 	}
 
 	id := r.URL.Query().Get("id")
-	if id == "" {
+	if id == "" || !isValidUUID(id) {
 		err := "vTPM launch request failed, id is required"
```

Comment:
How about including the potentially malformed uuid in this and later error strings?

Reply:
added.